config/manifests/kustomization.yaml

```diff
@@ -1,8 +1,28 @@
-# These resources constitute the fully configured set of manifests
-# used to generate the 'manifests/' directory in a bundle.
-resources:
-- bases/codeflare-operator.clusterserviceversion.yaml
-- ../default
-- ../scorecard
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+
+configMapGenerator:
+- name: codeflare-stack-config
+  envs:
+    - bases/params.env
+configurations:
+  - bases/params.yaml
+
+vars:
+- name: namespace
+  objref:
+    kind: ConfigMap
+    name: codeflare-stack-config
+    apiVersion: v1
+  fieldref:
+    fieldpath: data.namespace
+- name: odh_codeflare_operator_controller_image
+  objref:
+    kind: ConfigMap
+    name: codeflare-stack-config
+    apiVersion: v1
+  fieldref:
+    fieldpath: data.odh-codeflare-operator-controller-image
```

Comment:
The `odh-codeflare-operator-controller-image`  is being updated correctly for the version, and added to the configmap correctly, but it is not being set in the deployment anywhere, so we will need to adjust [here](https://github.com/project-codeflare/codeflare-operator/blob/main/config/manager/manager.yaml#L37) to use the image: 

```diff
-        image: controller:latest
+        image: $(odh_codeflare_operator_controller_image)

```


Namespace can also be removed from here

```diff
- - name: namespace
-  objref:
-    kind: ConfigMap
-    name: codeflare-stack-config
-    apiVersion: v1
-  fieldref:
-    fieldpath: data.namespace
```